### PR TITLE
coverage: Fix coverage for CI

### PR DESCRIPTION
**Pull request by phlax**
Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: test/per_file_coverage.sh [hunk 1]
```diff
@@ -48,6 +48,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/extensions/filters/listener/original_src:92.1"
 "source/extensions/filters/network/common:96.1"
 "source/extensions/filters/network/common/redis:96.24"
+"source/extensions/filters/network/http_connection_manager:96.4"
```
Comment (phlax) [inline test/per_file_coverage.sh:51]: @alyssawilk i guess this is less than ideal but im gonna land to get CI moving again

Comment (alyssawilk) [reply test/per_file_coverage.sh:51]: Do we know if it's intermittant or what?
https://storage.googleapis.com/envoy-postsubmit/main/coverage/source/extensions/filters/network/http_connection_manager/index.html looks like the cc file is 97.8 % and the .h file at 100%

Comment (phlax) [reply test/per_file_coverage.sh:51]: not sure - i was seeing mixed results at the time (iirc) - altho wasnt clear what was due to merging etc

struggling to rem now but i thought this was related to some removed code

Comment (phlax) [reply test/per_file_coverage.sh:51]: hmm - now im wondering if it was fixed here https://github.com/envoyproxy/envoy/commit/90ccaf931e8d2e7dcb0c91fa940040b17fbaae8e

Comment (alyssawilk) [reply test/per_file_coverage.sh:51]: oh probably.  anyway IMO fixing CI ASAP is the right thing to do, and thanks for both doing that, and ccing me so I didn't miss it in the backlog of holiday emails.  
https://github.com/envoyproxy/envoy/pull/24732 should hopefully fix

**Pull request merged**